packages/easy-coding-standard/build/target-repository/composer.json

```diff
@@ -5,6 +5,11 @@
     "require": {
         "php": ">=7.1"
     },
+    "require-dev": {
+        "phpunit/phpunit": "^9.5",
+        "symplify/easy-testing": "^9.3",
+        "symplify/smart-file-system": "^9.3"
+    },
```

Comment:
Test setup cannot be part of `composer.json`, this must happen in the workflow steps

Reply:
Why? I mean it's require-dev so it doesn't effect anyone who uses the package.

Reply:
Because that's part of test, not dependency

Reply:
It's a dependency of the test which is exactly the purpose of require-dev.